<?xml version="1.0" encoding="utf-8"?>
<launch>

	<node name = "hrc_2d_poser" pkg="third_arm" type="cog_hrc2d_apriltags_2d_ik.py" output ='screen'/>
  <node name = "hrc_motor_controller" pkg="third_arm" type="cog_hrc2d_master.py" output ='screen'/>
	<!--node name = "hrc_motor_controller" pkg="third_arm" type="cog_hrc2d_full_closed_loop.py" output ='screen'/-->
	<node name="hrc2d_data_logger" pkg="third_arm" type="reg_hrc2d_data_logger.py" output='screen' />

	<node pkg="rosbag" type="record" name="thirdarm_hrc2d_bag"
       args="record -o $(find third_arm)/scripts/HRC_2D_Task/Regression_Tests/Data/Bags/thirdarm_hrc2d_bag 
       				  /base_swivel_controller/state /vertical_tilt_controller/state /arm_extension_controller/state /wrist_controller/state /wrist_tilt_controller/state /gripper_controller/state  
                      /base_swivel_controller/command /vertical_tilt_controller/command
                      /arm_extension_controller/command /wrist_controller/command /wrist_tilt_controller/command /gripper_controller/command
                      /arm_command_state /base_pose /ee_pose /left_hand_pose /right_hand_pose /cup_pose 
                      /recognizer/output
                      /tag_detections
                      /hrc2d_task_number
                      /target_probs
                      /task_probs
                      /task_timeouts
                      "  
                      />


  	<!--node name = "hrc_motor_controller" pkg="third_arm" type="reg_hrc2d_speech_closed_loop.py" output ='screen'/-->
  	<!--node name="prob_plot_data_prep"  pkg="third_arm"  type="reg_hrc2d_prob_plotter.py" /-->
</launch>
